source/default/_patterns/00-protons/legacy/css/components/header/_utc_custom_header.css

```diff
@@ -1,158 +1,191 @@
+        /* color: blue; */
+        @apply text-utc-new-blue-500;
+    }
 }
 
 @media screen and (min-width: 992px) {
-  .block--system-branding-block img {
-    width: 60%;
-  }
+    .block--system-branding-block img {
+        width: 60%;
+    }
 }
 
 @media screen and (max-width: 992px) {
-  .header__container-wrapper--header__main {
-    -webkit-box-shadow: 0 3px 5px 0 rgba(0, 0, 0, 0.2);
-    box-shadow: 0 3px 5px 0 rgba(0, 0, 0, 0.2);
-  }
-  .header__container--boxed {
-    max-width: 100% !important;
-  }
+    .header__container-wrapper--header__main {
+        -webkit-box-shadow: 0 3px 5px 0 rgba(0, 0, 0, 0.2);
+        box-shadow: 0 3px 5px 0 rgba(0, 0, 0, 0.2);
+    }
+    .header__container--boxed {
+        max-width: 100% !important;
+    }
 }
 
 @media screen and (min-width: 992px) and (max-width: 1199.98px) {
-  .block--region-header-primary-menu > ul.menu > li > a {
-    font-size: 12px;
-    transition: all 0s ease-in-out;
-  }
+    .block--region-header-primary-menu>ul.menu>li>a {
+        font-size: 12px;
+        transition: all 0s ease-in-out;
+    }
 }
 
+
 /*new code*/
 
 @media screen and (max-width: 768px) {
-  .block--system-branding-block img {
-    width: 240px;
-  }
+    .block--system-branding-block img {
+        width: 240px;
+    }
 }
+
 @media screen and (max-width: 992px) {
-  .block--system-branding-block img {
-    width: 240px;
-  }
+    .block--system-branding-block img {
+        width: 240px;
+    }
 }
 
 @media screen and (max-width: 640px) {
-  .header__container {
-    padding-left: 0;
-  }
-  .block--region-logo {
-    position: relative;
-    width: fit-content !important;
-    overflow: hidden;
-    width: 37px !important;
-    margin-left: 0.3rem;
-  }
-  .header a.site-logo {
-    position: absolute;
-    width: 240px !important;
-    height: 30px !important;
-  }
+    .header__container {
+        padding-left: 0;
+    }
+    .block--region-logo {
+        position: relative;
+        width: fit-content !important;
+        overflow: hidden;
+        width: 37px !important;
+        margin-left: 0.3rem;
+    }
+    .region-logo {
+        padding-left: .75rem;
+    }
+    .header a.site-logo {
+        position: absolute;
+        width: 240px !important;
+        height: 30px !important;
+    }
+    ul.sf-menu ul li a {
+        font-size: 1rem;
+    }
 }
 
 @media screen and (max-width: 360px) {
-  .we-mega-menu-ul {
-    max-width: 175px;
-    margin-left: 6px;
-  }
-  .navbar-we-mega-menu.navbar .we-mega-menu-ul > li.we-mega-menu-li > a {
-    padding-left: 6px;
-    padding-right: 6px;
-  }
-  .navbar-we-mega-menu.navbar
-    .we-mega-menu-ul
-    > li.we-mega-menu-li[data-submenu='1']
-    > a:after {
-    margin-left: 4px;
-  }
-  .header__toggleable-account-menu {
-    display: none;
-  }
-  .header__user-nav .js-toggle-search {
-    padding-left: 12px;
-    padding-right: 0;
-  }
+    /*Drupal 8 mega menu is no longer used. Superfish is now in play.
```

Comment:
@UTCbridget let's do a pr that enables these classes again. Disabling those will break backwards compatibility if superfish does not work as expected once it reaches production. If superfish goes well we can remove them in a subsequent one.